src/lib/consensus/proof_of_stake.ml

```diff
@@ -1622,32 +1639,12 @@ module Data = struct
             , Epoch_data.Next_value_versioned.Value.Stable.V1.t
             , bool )
             Poly.Stable.V1.t
-          [@@deriving sexp, eq, compare, hash]
+          [@@deriving sexp, eq, compare, hash, yojson]
```

Comment:
Is there some danger if `last_vrf_output` is not opaque?

Reply:
What kind of danger? I'm encoding it in the Base64 format so it doesn't look weird in the log files.

Reply:
The old code made a point to JSON-encode it as `opaque`, so I'm supposing there's some crypto attack enabled by revealing it.

Reply:
It is part of the protocol state that is already public

Reply:
I think it was made opaque earlier because it did not offer any insight as such? Like the protocol state proof (and other fields) in `External_transition.t` and would make the logs huge

Reply:
Wikipedia says: 

```
VRF outputs can be published publicly without being subject to a preimage attack
```
so I guess it's OK!